### PR TITLE
added auto scroll for server selection list

### PR DESCRIPTION
**Pull request by ShehabAhmed-Gui**
I have added the auto scroll for servers selection list

### CODE REVIEW

File: client/ui/qml/Controls2/ListViewWithRadioButtonType.qml [hunk 1]
```diff
@@ -5,117 +5,118 @@ import QtQuick.Layouts
 import "TextTypes"
 
 ListView {
-    id: root
+  id: root
```
Comment (Nethius) [inline client/ui/qml/Controls2/ListViewWithRadioButtonType.qml]: Please return the indents to their previous state

File: client/ui/qml/Pages2/PageHome.qml [hunk 2]
```diff
@@ -16,362 +16,379 @@ import "../Config"
 import "../Components"
 
 PageType {
-    id: root
+  id: root
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageHome.qml]: Please return the indents to their previous state.

File: client/ui/qml/Pages2/PageStart.qml [hunk 3]
```diff
@@ -12,233 +12,259 @@ import "../Config"
 import "../Components"
 
 PageType {
-    id: root
+  id: root
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: Please return the indents to their previous state.

**Review by Nethius — CHANGES_REQUESTED** [review]

File: client/ui/qml/Pages2/PageStart.qml [hunk 4]
```diff
@@ -43,6 +43,8 @@ PageType {
         }
 
         function onClosePage() {
+            tabBar.isServerInfoShow = tabBarStackView.currentItem.objectName !== PageController.getPagePath(PageEnum.PageSettingsServerInfo)
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 5]
```diff
@@ -51,11 +53,14 @@ PageType {
 
         function onGoToPage(page, slide) {
             var pagePath = PageController.getPagePath(page)
+
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 6]
```diff
@@ -51,11 +53,14 @@ PageType {
...
             if (slide) {
                 tabBarStackView.push(pagePath, { "objectName" : pagePath }, StackView.PushTransition)
             } else {
                 tabBarStackView.push(pagePath, { "objectName" : pagePath }, StackView.Immediate)
             }
+
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 7]
```diff
@@ -109,14 +114,6 @@ PageType {
         }
     }
 
-    Connections {
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 8]
```diff
@@ -134,6 +131,7 @@ PageType {
             var pagePath = PageController.getPagePath(page)
             tabBarStackView.clear(StackView.Immediate)
             tabBarStackView.replace(pagePath, { "objectName" : pagePath }, StackView.Immediate)
+            tabBar.isServerInfoShow = false
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 9]
```diff
@@ -147,6 +145,7 @@ PageType {
         id: tabBar
 
         property int previousIndex: 0
+        property bool isServerInfoShow: false
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 10]
```diff
@@ -177,7 +176,7 @@ PageType {
         }
 
         TabImageButtonType {
-            isSelected: tabBar.currentIndex === 0
+            isSelected: tabBar.isServerInfoShow ? false : tabBar.currentIndex === 0
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageStart.qml [hunk 11]
```diff
@@ -211,7 +210,7 @@ PageType {
         }
 
         TabImageButtonType {
-            isSelected: tabBar.currentIndex === 2
+            isSelected: tabBar.isServerInfoShow ? true : tabBar.currentIndex === 2
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageStart.qml]: I don't think these changes are needed in this PR

File: client/ui/qml/Pages2/PageHome.qml [hunk 12]
```diff
@@ -242,6 +243,13 @@ PageType {
                 implicitHeight: parent.height - serversMenuHeader.implicitHeight
                 clip: true
 
+                function autoScroll(){
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageHome.qml]: Please add indents as in the rest of the project source files

**Review by Nethius — CHANGES_REQUESTED** [review]

File: client/ui/qml/Pages2/PageHome.qml (hunk 12 above)
Comment (ShehabAhmed-Gui) [reply client/ui/qml/Pages2/PageHome.qml]: I think I understand the indents structure of the application now, please re-review, if there is a problem. please clarify it.

**Pull request closed without merging**